docs/dev/nightly_data_builds.rst

```diff
@@ -14,12 +14,29 @@ pushes a Docker image with PUDL installed to `Docker Hub <https://hub.docker.com
 and deploys the image as a container to a GCE instance. The container runs the ETL and
 tests, then copies the outputs to a public AWS S3 bucket for distribution.
 
+
 Breaking the Builds
 -------------------
 The nightly data builds based on the ``main`` branch are our comprehensive integration
-tests. When they pass, we consider the results fit for public consumption.  The builds
-are expected to pass. If they don't then someone needs to take responsibility for
-getting them working again with some urgency.
+tests. When they pass, we consider the results fit for public consumption.
+
+When they don't pass, we need to fix them.
+
+Every morning, someone from inframundo will check the #pudl-deployments slack
```

Comment:
@catalyst-cooperative/inframundo is it ok to sign us up for this? I think it's mostly been @zaneselvans and @bendnorman checking in the past. I've been doing it in the new year. We could set up a rotation if we want.

Reply:
I think this is a realistic description of this team's responsibilities, even if we wind up delegating the fix to someone outside of inframundo.